### DISCUSSION TITLE
Ability to set Tailwind Prefix in components.json

### DISCUSSION THREAD

**Discussion by tobischenk** [unovue/shadcn-vue#444] [Q&A] [answered]
Hi together,

I was wondering if supporting tailwindcss prefixes in up for debate, since in shadcn-ui it is implented. Are there any plans to include the cli improvements from this [pull request](https://github.com/radix-vue/shadcn-vue/pull/324), which was merged on 21st of December last year.

If not I can look into providing a pr on my own, but I have not yet worked in this repo, so it will probably take some time.

Thanks in advance!

**Comment by sadeghbarati** [comment, marked as answer]
Hi @tobischenk 

Please look at CLI source, it has prefix implementation but its not completed that's why Zernonia hide it in CLI init 


You can check the source of CLI if you interested to contribute ❤️

Also for context you can check this PR https://github.com/radix-vue/shadcn-vue/pull/324#issuecomment-1977926222

**Reply by tobischenk** [reply]
Thanks for the quick reply! I will have a look at it and see if I can help by contributing. I am not sure if I find the time for it,  but I will try

**Reply by tobischenk** [reply]
And i wanted to like this this pr from [shadcn-ui](https://github.com/shadcn-ui/ui/pull/770) and not the one from this repo. I will have a look at changes made there and then possibly make a pr to apply those to this project as well 👍

**Reply by tobischenk** [reply]
I don't think I i will find to implement this myself anytime soon. I am very sorry about that. For now I just use a tool to add prefixes to the added Components, which can be found here

https://github.vue.tailwind-prefix.cbass.dev/

As soon as i find time I will look into this and see. if its still needed.
